doc/Learn/Tutorials/Navigation/Advanced/HowTo-UseNavigationView.md

```diff
@@ -3,31 +3,161 @@ uid: Learn.Tutorials.Navigation.Advanced.NavigationView
+  ```
+
+* Add a `NavigationView` control to the `Grid`. This will be the menu for the app.
+
+  ```xml
+  <NavigationView Grid.Row="1">
+      ...
+  </NavigationView>
+  ```
+
+* Add a `Grid` element to the `NavigationView`. This will be the container for the content area.
+
+  ```xml
+  <Grid Grid.Row="1">
+      ...
+  </Grid>
+  ```
+
+### 3. Add the navigation view items
+
+* Add the `NavigationView.MenuItems` collection to the `NavigationView` and add a `NavigationViewItem` for each view you want to navigate to.
+
+  ```xml
+  <NavigationView.MenuItems>
+      <NavigationViewItem Content="One" />
+      <NavigationViewItem Content="Two" />
+      <NavigationViewItem Content="Three" />
+  </NavigationView.MenuItems>
+  ```
+
+### 4. Add the navigation view content
+
+* Inside the `Grid` element of the `NavigationView`, add a `Grid` element to represent the content of each view you can to navigate to.
+
+  ```xml
+  <Grid Grid.Row="1">
+      <Grid>
+          <TextBlock Text="One"
+                    FontSize="24"
+                    HorizontalAlignment="Center"
+                    VerticalAlignment="Center" />
+      </Grid>
+      <Grid>
+          <TextBlock Text="Two"
+                    FontSize="24"
+                    HorizontalAlignment="Center"
+                    VerticalAlignment="Center" />
+      </Grid>
+      <Grid>
+          <TextBlock Text="Three"
+                    FontSize="24"
+                    HorizontalAlignment="Center"
+                    VerticalAlignment="Center" />
+      </Grid>
+  </Grid>
+  ```
+
+### 5. Set up navigation
+
+* Components with navigation enabled will have their visual trees considered by the navigation service when a request is made. The matching **region** will be shown and the rest will be hidden.
+
+* Add the `uen:Region.Attached` attached property to each of the following elements:
+
+    * The `Grid` element that contains the `NavigationView` and the `Grid` element that contains the content area.
+    * The `NavigationView` element.
+    * The `Grid` element that contains the content area.
+
+  ```xml
+  uen:Region.Attached="True"
+  ```
+
+* Setting this to `True` will enable the navigation extensions for the element.
+
+* Add the `uen:Region.Name` attached property to each of the `NavigationViewItem` elements. This will define the name of the view that the `NavigationViewItem` will navigate to.
+
+  ```xml
+  uen:Region.Name="One|Two|Three"
+  ```
+
+* Add the `uen:Region.Navigator` attached property to the `Grid` element that contains the content area. This will set the type of navigation to adjust the visibility of the content area's children.
+
+  ```xml
+  uen:Region.Navigator="Visibility"
+  ```
+
+* Add the `uen:Region.Name` attached property to each of the `Grid` elements that contain the content area. This will define the name of the view that the `Grid` will represent.
+
+  ```xml
+  uen:Region.Name="One|Two|Three"
```

Comment:
Again, show one example and then the full XAML after setting up the entire set of regions